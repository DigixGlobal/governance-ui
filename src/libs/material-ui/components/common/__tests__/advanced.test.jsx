import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Advanced from '../advanced';

describe('<Advanced />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <Advanced><div>Test</div></Advanced>,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
