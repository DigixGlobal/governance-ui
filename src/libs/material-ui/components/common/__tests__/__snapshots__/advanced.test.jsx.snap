// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Advanced /> renders correctly 1`] = `
<FormField>
  <a
    onClick={[Function]}
    role="button"
    style={
      Object {
        "cursor": "pointer",
      }
    }
    tabIndex={0}
  >
    Advanced
     
    <Icon
      as="i"
      name="caret right"
    />
  </a>
</FormField>
`;
